components/rhel/misc/hyperv/rhel-ose-hyperv/README.rst

```diff
@@ -0,0 +1,61 @@
+Vagrantfile to setup  OpenShift Enterprise
+==========================================
+
+This Vagrantfile sets up the CDK for development with OpenShift. This Vagrantfile provisions an instance of `OpenShift Enterprise <https://www.openshift.com/enterprise/whats-new.html>`_.
+
+**Note:** This Vagrantfile expects an external type virtual switch for HyperV. Please add a new external type virtual network switch to the HyperV setup if you do not have it.
+
+
+QuickStart
+----------
+
+1. Get latest CDK box (Using Developer Subscription).
```

Comment:
@LalatenduMohanty : created the issue here https://github.com/projectatomic/adb-atomic-developer-bundle/issues/444